lib/scalingo/auth/user.rb

```diff
@@ -27,5 +27,18 @@ def update(payload, headers = nil, &block)
 
       unpack(:user) { response }
     end
+
+    def stop_free_trial(headers = nil, &block)
+      data = nil
+
+      response = connection.post(
+        "users/stop_free_trial",
+        data,
+        headers,
+        &block
```

Comment:
Can you quickly what the `headers` and `block` variable are for? My reflex would be to have the `stop_free_trial` method to not take any argument.

Reply:
It's the standard skeleton for methods. They are all overridable in the same manner, either by passing additional headers, or by manipulating the faraday request via a block. All methods ends with those two params and forwards them, it allows raw-er access in case the lib is missing something, outdated, etc

Reply:
All of this is in the readme